build/templates/functions.rst/lock_session_wrapper.rst.mako

```diff
@@ -0,0 +1,47 @@
+<%page args="function, config, method_template, indent"/>\
```

Comment:
Since the method this is used for is `lock`, I think the filename should be `lock.rst.mako` (I'm not sure what "wrapper" buys us).

This applies to the rest of the new mako templates.